components/oauth-extensions/dynamic-client-manager/src/main/java/org/wso2/carbon/identity/oauth/extension/impl/ClientRegistrationServiceImpl.java

```diff
@@ -232,4 +250,45 @@ public OAuthApplicationInfo createOAuthApplication(
         return null;
     }
 
+    public void unregisterApplication(String userId, String applicationName, String consumerKey)
```

Comment:
This class is removed and the function is introduced as an util method in DynamicClientRegistrationUtil class in commit [1].
[1] https://github.com/wso2/carbon-device-mgt/commit/8073d46f58c8eede4d1960b026cc32b7b23b28cb